README.md

```diff
@@ -3,44 +3,216 @@
+- [Deploying the resource aggregator for ODIM (ODIMRA)](#deploying-the-resource-aggregator-for-odim--odimra-)
+  * [1. Setting up environment](#1-setting-up-environment)
+    + [Preparing the odim-controller node](#preparing-the-odim-controller-node)
+    + [Installing Docker](#installing-docker)
+    + [Building Docker images](#building-docker-images)
+    + [Deploying Kubernetes on the odim-controller node](#deploying-kubernetes-on-the-odim-controller-node)
+    + [Copying Docker images to the master and worker nodes](#copying-docker-images-to-the-master-and-worker-nodes)
+  * [2. Installing ODIMRA](#2-installing-odimra)
     + [Default user credentials for ODIMRA and GRF Plugin](#default-user-credentials-for-odimra-and-grf-plugin)
 - [Modifying default configuration parameters for the resource aggregator](#modifying-default-configuration-parameters-for-the-resource-aggregator)
+- [Updating the Kubernetes Configuration file](#updating-the-kubernetes-configuration-file)
+  * [**Sample Config file**](#--sample-config-file--)
+- [Setting proxy](#setting-proxy)
+- [Setting up NTP client](#setting-up-ntp-client)
 - [Configuring proxy for Docker](#configuring-proxy-for-docker)
+- [Resetting Kubernetes](#resetting-kubernetes)
+- [Removing a node from the Kubernetes cluster](#removing-a-node-from-the-kubernetes-cluster)
+- [Uninstalling ODIMRA](#uninstalling-odimra)
+
+
    
 
 
 
 # Deploying the resource aggregator for ODIM (ODIMRA)
 
-## 1. Setting up OS and Docker environment
+## 1. Setting up environment
+
+This procedure provides step-by-step instructions on how to deploy Docker and Kubernetes. You will require a minimum of two machines - one as a deployment node where odim-controller is installed and the others as master and worker nodes where ODIMRA is deployed. 
 
 **Prerequisites**
-------------------
-- Ensure that the Internet is available. If your system is behind a corporate proxy or firewall, set your proxy configuration. To know how to set proxy, see information provided at `https://www.serverlab.ca/tutorials/linux/administration-linux/how-to-set-the-proxy-for-apt-for-ubuntu-18-04/`.  
+-----------------
+- Ensure that the odim-controller node has a minimum RAM of 8GB (8192MB), four cups, and 100GB of Hdd.
+
+- Ensure that the master and worker nodes have a minimum RAM of 16GB (16384MB), four cups, and 150GB of Hdd.
+
+- Ensure that the Internet is available on all the machines. If the systems are behind a corporate proxy or firewall, [set your proxy configuration on all the machines](#setting-proxy).  
+
+- Ensure to create same non-root username and password on all the machines and login with the same.
 
 - Ensure not to create `odimra` user during the installation of the VM.
 
+- Ensure that the FQDN entry in the `/etc/hosts` file is same on all the machines.
+
+- Ensure that time across the machines are in sync.
+
+- Ensure to [set the NTP client on the machines](#setting-up-ntp-client) to sync with NTP server in your environment.
```

Comment:
we do not set up NTP client. The timeanddate service should be configured to get sync time with any NTP server used in their setup. And how can we do this before the OS is installed ?